### ISSUE TITLE
Hide confidential information

### ISSUE THREAD

**Issue by heyjoja** [maptiler/tileserver-gl#713]
Is it possible to block this section to others users? For example, I need to maintain hide this information because it is information about my clients, maybe hide the links, or the list.

Thank you so much

![Screenshot 2023-01-13 at 9 25 49 PM](https://user-images.githubusercontent.com/7897954/212446766-a670782d-a668-4a75-99d8-315a4a4c72da.png)

**Comment by acalcutt** [comment]
I don't think there is anything right now in tileserver that can block that page. Typically you run it behind something like haproxy/nginx/apache and you may have some options to restrict the root page at that level

**Comment by acalcutt** [comment]
If you are running locally or building your own docker you could edit
https://github.com/maptiler/tileserver-gl/blob/master/public/templates/index.tmpl to hide the information

**Comment by boldtrn** [comment]
There is a config setting for this:

`"frontPage": false,`

**Comment by heyjoja** [comment]
Hi @boldtrn, but in the same file for the layers?

**Comment by heyjoja** [comment]
@acalcutt thank you, I edit the page to hide the information, and it's working but the option to disable the frontpage is so cool, do you know something about it?

**Comment by boldtrn** [comment]
Yes, this is for the config file. This is in the readme: https://tileserver.readthedocs.io/en/latest/config.html#frontpage

**Comment by acalcutt** [comment]
I didn't know about that option, but it sounds like what you wanted. let us know if it works
